torch/_dynamo/variables/ctx_manager.py

```diff
@@ -521,24 +521,28 @@ def exit(self, tx, *args):
         )
         self.state.cleanup_assert()
 
-    def module_name(self):
-        return "torch." + str(self.device)
-
-    def fn_name(self):
-        return "stream"
+    def reconstruct(self, codegen):
+        codegen.load_import_from("torch", "cuda.stream")
+        assert len(self.target_values) == 1
+        codegen(self.target_values[0])
+        codegen.extend_output(create_call_function(1, False))
```

Comment:
Sorry, this still seems like it's creating a new stream?

Reply:
See PR desc, this is a stream context, not a stream, and it was already doing this, but incorrectly.